sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/AzureOpenAIOperationState.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+package com.azure.ai.openai.implementation.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** The state of a job or item. */
+public final class AzureOpenAIOperationState extends ExpandableStringEnum<AzureOpenAIOperationState> {
```

Comment:
diff to models
https://github.com/Azure/azure-sdk-for-java/blob/4f129e536ceb079f89b0b682e80cc25c0df5dead/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/AzureOpenAIOperationState.java#L29-L34